packages/hello-gsm-admin/src/components/ContentBox/index.tsx

```diff
@@ -56,134 +57,91 @@ const ContentBox: React.FC<ContentType> = ({
   }, []);
 
-  useEffect(() => {
-    setDocumentReception(isDocumentReception);
-  }, [isDocumentReception]);
-
-  useEffect(() => {
-    setFirstResult(
-      !isFirstResult ? '미정' : firstResultScreening ? '합격' : '불합격',
-    );
-  }, [isFirstResult]);
-
-  useEffect(() => {
-    if (!showScoreModal) {
-      if (modalRegistrationNumber === registrationNumber) {
-        setScore(scoreModalValue);
-      }
-    }
-  }, [showScoreModal]);
-
-  useEffect(() => {
-    setScore(
-      parseFloat(application_score?.personalityEvaluationScore ?? '') || null,
-    );
-  }, [application_score]);
-
-  const documentSubmission = async () => {
-    // 1차 서류제출 여부를 할당하는 기간
-    if (isStartFirstResult) {
-      return toast.error('서류제출 여부 할당 기간이 아닙니다.');
-    }
-    const data = {
-      registrationNumber: registrationNumber,
-    };
-    try {
-      await application.document(data);
-      setDocumentReception(documentReception => !documentReception);
-    } catch (error: any) {
-      // accessToken 없을 시에 accessToken 발급 후 서류 제출 여부 요청
-      if (error.response.status === 401) {
-        try {
-          // accessToken 발급
-          await auth.refresh();
-          documentSubmission();
-        } catch (error) {
-          console.log(error);
-        }
-      } else {
-        console.log(error);
-      }
-    }
-  };
-
   const resultStyle = (result: '미정' | '합격' | '불합격') => {
     switch (result) {
       case '미정':
         return css`
-          background: #625e6f;
-          color: rgba(31, 31, 31, 0.86);
+          color: #9e9e9e;
           cursor: default;
         `;
       case '합격':
         return css`
-          background: #19baff;
-          color: #ffffff;
+          color: #2174d8;
           cursor: default;
         `;
       case '불합격':
         return css`
-          background: #ff4747;
-          color: #ffffff;
+          color: #ff000f;
           cursor: default;
         `;
     }
   };
 
-  const scoreButtonClick = () => {
-    if (
-      // 2차 성적을 입력하는 기간
-      isDuringFinalResult
-    ) {
-      return toast.error('2차 성적 입력 기간이 아닙니다.');
-    }
-    if (firstResult === '불합격') {
-      return toast.error('불합격자는 2차 성적 입력이 불가능합니다.');
-    }
-    setModalRegistrationNumber(registrationNumber);
-    setModalName(name);
-    setShowScoreModal();
-    setScoreModalValue(score);
-  };
-
   return (
     <S.ContentBox>
       <S.Content>
-        <S.RegistrationNumber>{applicationIdx}</S.RegistrationNumber>
+        <S.RegistrationNumber>
+          {String(applicationIdx).padStart(4, '0')}
+        </S.RegistrationNumber>
+        <S.isDocumentReception>
+          <S.DocumentReceptionText documentReception={documentReception}>
+            {documentReception ? '· 제출' : '· 미제출'}
+          </S.DocumentReceptionText>
+        </S.isDocumentReception>
         <S.Name>{name}</S.Name>
         <S.Screening>{screening}</S.Screening>
         <S.SchoolName>
           {schoolName !== 'null' ? schoolName : '검정고시'}
```

Comment:
```suggestion
          {schoolName !== null ? schoolName : '검정고시'}
```

`null`이랑 비교해야할 것 같습니다.

Reply:
넵 `'null'` 을 비교하는 다른 부분들도 함께 수정하여 커밋 하겠습니다!

Reply:
40f1519e857f15d7dbdbd33b0de6dd8ebded26d4
수정하였습니다!